stages/01_adrecs.py

```diff
@@ -1,21 +1,45 @@
 # Imports
 # -------
-import re
 import os
 import gzip
+import click
 import pandas as pd
 import urllib.request
 
+@click.command()
+@click.option('--download_data', default=False, help='Run the downloading of the ADRECS data')
+@click.option('--process_data', default=False, help='Process the Data into parquet files')
```

Comment:
Maybe we can chat about making a command line pipeline for the bricks if we need too. For now, since most bricks are not built in that way I don't think I should start with a different way. 

Resolved in this commit to split the files.

57a5a6bf583488af0ef0743a5a01181957ba812f